lib/code_climate/test_reporter/version.rb

```diff
@@ -1,5 +1,5 @@
 module CodeClimate
   module TestReporter
-    VERSION = "1.0.7".freeze
+    VERSION = "1.0.8".freeze
```

Comment:
Feel free to edit the PR as you wish. There is some date in the changelog that is most probably the one for when the version is released.

```
Lock simplecov to supported versions - `>= 0.10` and `<= 0.13`
```